src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -58,5 +59,22 @@ private void ExecuteSearchElement(ListBoxItem listBoxItem)
             }
         }
 
+        private void OnInCanvasSearchControlVisibilityChanged(object sender, DependencyPropertyChangedEventArgs e)
+        {
+            // If visibility  is false, then stop processing it.
+            if (!(bool)e.NewValue)
+                return;
+
```

Comment:
If you asked to add possibility to selected test, I thought it's better to add focus as well.
Link to YouTrack:
[MAGN-7406](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7406) Incanvas search should have focus on initiation